File: 2466/Sinuca.cpp
#include <bits/stdc++.h>
using namespace std;


int main(){
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n; cin >> n;
    vector<vector<int>> balls;
    vector<int> first;
    vector<int> rows;
    
    for (int i = 0; i < n; i++){
        
        int value; cin >> value;
        first.push_back(value);
    }

    balls.push_back(first);
    for (int j = 1; j <= n; j++){
        for (int i = 1; i <= n - j; i++){
            //cout << j << " " << i << endl;
            if (balls[j-1][i-1] == 1 && balls[j-1][i] == 1){
                //cout << "Entre1\n";
                rows.push_back(1);
            }
            else if (balls[j-1][i-1] == -1 && balls[j-1][i] == -1){
                rows.push_back(1);
                //cout << "Entrei2\n";
            }
            else
                rows.push_back(-1);
            
        }
        balls.push_back(rows);
        rows.clear();
    }
    
    if (balls[n-1][0] == -1)
        cout << "branca\n";
    else
        cout << "preta\n";
}

File: 2466/Sinuca.py
n = int(input())

first = input().split()


rows = []
rows.append(first)

for j in range(1, n):
    rest = []
    for i in range(1, n-j+1):
        
        #print(str(j) + " " + str(i))
        if rows[j-1][i-1] == '1' and rows[j-1][i] == '1':
            rest.append('1')
        elif rows[j-1][i-1] == '-1' and rows[j-1][i] == '-1':
            rest.append('1')
        else:
            rest.append('-1')


    rows.append(rest)
    
    #print(rows)


if rows[n-1][0] == '-1':
    print("branca")
else:
    print("preta")
